docs/src/main/paradox/release-notes/2.0.x.md

```diff
@@ -16,6 +16,49 @@ The Alpakka Kafka 2.0 series features
 * Removal of some API which was deprecated since 1.0.0 or earlier.
 * Internal filter to avoid emitting records of revoked partitions [#946](https://github.com/akka/alpakka-kafka/issues/946) and [#992](https://github.com/akka/alpakka-kafka/issues/992)
 
+# 2.0.4
+
+Released: 2020-07-22
+
+## Notable changes since 2.0.3
+
+- Emit offset batches on timer [#1160](https://github.com/akka/alpakka-kafka/issues/1160) by [@jyates](https://github.com/jyates)
+- Do not commit offsets for unassigned partitions [#1123](https://github.com/akka/alpakka-kafka/issues/1123) by [@jyates](https://github.com/jyates)
+- Use bulk update API for shard allocation [#1139](https://github.com/akka/alpakka-kafka/issues/1139) by [@chbatey](https://github.com/chbatey)
+- Add produce consume check to KafkaContainerCluster [#1131](https://github.com/akka/alpakka-kafka/issues/1131) by [@seglo](https://github.com/seglo)
+- Deprecate embedded kafka support [#1136](https://github.com/akka/alpakka-kafka/issues/1136) by [@seglo](https://github.com/seglo)
+
+## Alpakka Kafka core
+
+- Emit offset batches on timer [#1160](https://github.com/akka/alpakka-kafka/issues/1160) by [@jyates](https://github.com/jyates)
+- Do not commit offsets for unassigned partitions [#1123](https://github.com/akka/alpakka-kafka/issues/1123) by [@jyates](https://github.com/jyates)
+
+## Alpakka Kafka Akka Cluster Sharding
+
+- Use bulk update API for shard allocation [#1139](https://github.com/akka/alpakka-kafka/issues/1139) by [@chbatey](https://github.com/chbatey)
+- Don't use ctx.log in KafkaClusterSharding [#1138](https://github.com/akka/alpakka-kafka/issues/1138) by [@chbatey](https://github.com/chbatey)
+
+## Alpakka Kafka Testkit
+
+- Deprecate embedded kafka support [#1136](https://github.com/akka/alpakka-kafka/issues/1136) by [@seglo](https://github.com/seglo)
+- Replace embedded kafka with schema registry with testcontainers [#1135](https://github.com/akka/alpakka-kafka/issues/1135) by [@seglo](https://github.com/seglo)
+- Add produce consume check to KafkaContainerCluster [#1131](https://github.com/akka/alpakka-kafka/issues/1131) by [@seglo](https://github.com/seglo)
+
+The milestone contains everything [*closed in 2.0.4*](https://github.com/akka/alpakka-kafka/issues?q=is%3Aclosed+milestone%3A2.0.4).
+
+Since 2.0.3 Alpakka Kafka has received contributions by:
+
+| Author | Commits | Lines added | Lines removed |
+| ------ | ------- | ----------- | ------------- |
+| [<img width="20" alt="scala-steward" src="https://avatars1.githubusercontent.com/u/43047562?v=4&amp;s=40"/> **scala-steward**](https://github.com/scala-steward) | 10 | 11 | 11 |
```

Comment:
maybe leave `scala-steward` off the list?

Reply:
I was wondering about that.  Is that a standard policy?

Reply:
I'll remove it and add an updates section with all of the scala-steward merged PRs.